frontend/test/metabase/scenarios/collections/bookmarks.cy.spec.js

```diff
@@ -0,0 +1,107 @@
+import { restore, sidebar } from "__support__/e2e/cypress";
+import { USERS } from "__support__/e2e/cypress_data";
+import { getSidebarSectionTitle as getSectionTitle } from "__support__/e2e/helpers/e2e-collection-helpers";
+
+const adminFullName = USERS.admin.first_name + " " + USERS.admin.last_name;
+const adminPersonalCollectionName = adminFullName + "'s Personal Collection";
+
+describe("Bookmarks in a collection page", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("cannot add bookmark to root collection", () => {
+    cy.intercept("GET", "/api/collection/root/items?**").as(
+      "fetchRootCollectionItems",
+    );
+
+    cy.visit("/collection/root");
+
+    cy.wait("@fetchRootCollectionItems");
```

Comment:
```suggestion
    cy.wait(["@fetchRootCollectionItems", "@fetchRootCollectionItems"]);
```

There are two such requests that we need to wait for. 
![image](https://user-images.githubusercontent.com/31325167/159554815-ef8b2c51-09a3-4c35-a2f1-f4dbc8dbcfa9.png)